modules/page/html/render/message.js

```diff
@@ -229,15 +249,24 @@ exports.create = function (api) {
     api.sbot.async.get({ id, private: true, unbox }, (err, value) => {
       if (err) return cb(err)
       var msg = { key: id, value }
-      if (isBlog(msg)) {
-        Blog(api.sbot.obs.connection).async.get(msg, (err, result) => {
-          if (err) return cb(err)
-          msg.body = result.body
-          cb(null, msg)
+
+      var me = api.keys.sync.id()
+      onceTrue(api.sbot.obs.connection, sbot => {
+        sbot.patchwork.contacts.isBlocking({ source: me, dest: value.author }, (err, blocking) => {
```

Comment:
What does this do? I'd love some comments here explaining the behavior that this creates.

Reply:
I think this whole blob of code was me thinking a bug (direct links to blocked people's posts still rendered) was my fault and trying to fix it. Turned out it was in all along, I think, but now it's fixed anyway hehe.

If you're asking about the whole onceTrue thing that's a piece of code I found somewhere else, I needed an sbot connection (still not clear on why haha) to get the isBlocking function.

Reply:
Satisfying explanation? :P Want any of it as comments in the code?

Reply:
Nope, I think this bit makes sense to me now (although comments are always appreciated!). :+1: